spec/index.md

```diff
@@ -4203,6 +4211,8 @@ New_canister_history = {
 
 if A.settings.log_visibility is not null:
   New_canister_log_visibility = A.settings.log_visibility
+  if New_canister_log_visibility is AllowList:
+    New_canister_log_allowlist = A.settings.log_visibility ?? how to get allowlist?
```

Comment:
@mraszyk could you please suggest on how to write down updating allowlist in this notation?

Reply:
Removed `New_canister_log_allowlist`.